src/otx/core/model/classification.py

```diff
@@ -132,7 +306,7 @@ def _convert_pred_entity_to_compute_metric(
 
     def forward_for_tracing(self, image: Tensor) -> Tensor | dict[str, Tensor]:
         """Model forward function used for the model tracing during model exportation."""
-        return self.model.forward(image, mode="tensor")
```

Comment:
It's just a question. Is it ok to remove `mode="tensor"`?

Reply:
`mode` is currently out of place in core because it relies on `ImageClassifier`.